### DISCUSSION TITLE
Giada wont find VST Plugins during scan...

### DISCUSSION THREAD

**Discussion by capncruncky** [monocasual/giada#490] [Q&A] [answered]
Hey! I need some help debugging why Giada wont find all of my VST Plugins during the Configuration scan....
I hold all of my VST Plugins in a master folder: 'VST MASTER'
In the Configuration/Plugins tab, I select this folder as the "Plugins folder" and then click "SCAN" - no plugins are found. I've removed the 'requested directories' string from the giada.conf file and attempted to re-enter the location in the Configuration/Plugins tab with no success. 

It is able to find one plugin with a '.vst3' extension...but most of my plugins are .dll
Is there a requirement/limitation for only '.vst3' plugins in Giada? 

Any help is greatly appreciated! I'm looking forward to solving this so I can actually use the software...

-B

edit:
I'm using Windows 10 and the rest of the software appears to be working correctly. I'm able to successfully map to a samples directory containing samples...

**Comment by capncruncky** [comment, marked as answer]
Ok so upon further research I discovered my VST3 files were stored in the default location: C:\Program Files\Common Files\VST3\
I was able to load the VST3 plugins from my collection that had vst3 files in that dir.

**Reply by CTBlaze** [reply]
Do you also have licensed some products which still only offered as VST2? I think giada makers should reconsider supporting VST2, or even provide several pre-compiles to leave the choice to user. (I am too lazy to compile myself haha)
